File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        Map<Integer, Integer> counts = new HashMap<>();
        int[] result = new int[k];
        List<int[]> freqs = new ArrayList<>();
        for (int num : nums) {
            counts.put(num, counts.getOrDefault(num, 0) + 1);
        }
        for (Map.Entry<Integer, Integer> entry : counts.entrySet()) {
            freqs.add(new int[] {entry.getValue(), entry.getKey()});
        }
        freqs.sort((a, b) -> b[0] - a[0]);
        int[] res = new int[k];
        for (int i = 0; i < k; i++) {
            res[i] = freqs.get(i)[1];
        }
        return res;
    }
}

File: Data Structures & Algorithms/counting-bits/submission-0.py
class Solution:
    def countBits(self, n: int) -> List[int]:
        counts = [0] * (n + 1)
        for i in range(n + 1):
            m = i
            while m > 0:
                if m & 1 == 1:
                    counts[i] += 1
                m = m >> 1
        return counts